code/modules/admin/verbs/adminhelp.dm

```diff
@@ -45,7 +45,7 @@ GLOBAL_LIST_INIT(adminhelp_ignored_words, list("unknown","the","a","an","of","mo
 			var/active_admins = admincount[1]
 
 			log_admin("[selected_type]: [key_name(src)]: [msg] - heard by [active_admins] non-AFK admins.")
-			SSdiscord.send2discord_simple_noadmins("[selected_type] from [key_name(src)]: [msg]", check_send_always = TRUE)
+			SSdiscord.send2discord_simple_noadmins("**\[Adminhelp]**  [key_name(src)]: [msg]", check_send_always = TRUE)
```

Comment:
```suggestion
			SSdiscord.send2discord_simple_noadmins("**\[Adminhelp]** [key_name(src)]: [msg]", check_send_always = TRUE)
```
Remove double space (Making it a suggestion so I can commit it without using the web editor)